src/util/util.ts

```diff
@@ -89,3 +89,21 @@ export const getEtasuUrl = () => {
     useIntermediary ? intermediaryBaseUrl : nonIntermediaryBaseUrl
   }/4_0_0/GuidanceResponse/$rems-etasu`;
 };
+
+export const extractRemsAdminBaseUrl = (questionnaireUrl: string): string | null => {
+  try {
+    const url = new URL(questionnaireUrl);
+    return `${url.protocol}//${url.host}`;
```

Comment:
What if there is more to the base url thank just the protocol and host? something like:
http://remshost.com/fhir/R4/Questionnaire
the fhir/R4 would be missing?